tf/environments/dev/main.tf

```diff
@@ -285,6 +285,10 @@ module "oonidataapi" {
   )
 }
 
+module "ooni_dev_user" {
```

Comment:
I think in terms of naming we should follow the pattern we have for other modules and clarify that this particular account is only to be used for github.

To this end I think we should call it `oonidevops_github`.

I also think we can probably just put all of this code inside of the `adm_iam_roles` module since it fits naturally inside of the base `adm_` tasks that are needed for everything to function.

Reply:
Actually scratch that, we might want to create this role only for certain environments, like test and dev, but probably not for prod or at least have the possibility to not create it in that case.

I think leaving it as it's own module is good, however we should rename the module to `oonidevops_github_user`.